webapp/src/components/sidebar_right/github_items.tsx

```diff
@@ -56,6 +129,14 @@ function GithubItems(props) {
                 </strong>);
         }
 
+        let titleText = '';
+        if (item.title) {
+            titleText = item.title;
+        } else if (item.subject?.title) {
+            titleText = item.subject.title;
+        }
+
+        let title:JSX.Element | null = null;
```

Comment:
0/5, then you can remove the `else` below

```suggestion
        let title: React.ReactNode = titleText;
```